WordPress/Classes/ViewRelated/Comments/CommentDetailViewController.swift

```diff
@@ -58,6 +86,46 @@ private extension CommentDetailViewController {
         tableView.tableFooterView = UIView(frame: .zero)
     }
 
+    func configureRows() {
+        rows = [.header]
+    }
+
+    // MARK: Cell configuration
+
+    func configureHeaderCell() {
+        // TODO: detect if the comment is a reply.
+
+        headerCell.textLabel?.text = .postCommentTitleText
+        headerCell.detailTextLabel?.text = comment.titleForDisplay()
+    }
+
+    // MARK: Actions and navigations
+
+    func navigateToPost() {
+        guard let blog = comment.blog,
+              let siteID = blog.dotComID,
+              blog.supports(.wpComRESTAPI) else {
+            viewPostInWebView()
+            return
+        }
+
+        let readerViewController = ReaderDetailViewController.controllerWithPostID(NSNumber(value: comment.postID), siteID: siteID, isFeed: false)
+        navigationController?.pushFullscreenViewController(readerViewController, animated: true)
```

Comment:
As I was testing on iPad, I noticed an odd behavior which I think is related to `pushFullscreenViewController`. The `readerViewController` tries to appear full screen, then jumps down to split view. I see this is simply copied behavior from `CommentViewController`, and I did confirm this is an existing issue in the app store version. But maybe we can fix it with this feature? (at some point, not in this PR)

Reply:
Yeah, that makes sense. I'll note this down in #17087 and address this in a separate PR. Thanks! 🙂